modules/monitoring/templates/dashboard/core/utils.js

```
@@ -60,9 +61,82 @@ function _normalizeHardwareFamily(unit) {
+function _readStoredKey(name) {
+    try {
+        if (typeof localStorage !== 'undefined' && localStorage && typeof localStorage.getItem === 'function') {
+            return localStorage.getItem(name);
+        }
+    } catch (_) {
+        return null;
+    }
+    return null;
+}
+
+function _writeStoredKey(name, value) {
```

ESLint

[error] Function '_writeStoredKey' has a complexity of 7. Maximum allowed is 5.

(complexity)